src/synchronizeContacts.js

```diff
@@ -130,13 +130,18 @@ const determineActionOnCozy = (
 const determineActionOnGoogle = (cozyContact, contactAccountId) => {
   const syncInfo = get(cozyContact, `cozyMetadata.sync.${contactAccountId}`)
   const isTrashedOnCozy = cozyContact.trashed
+  const lastUpdatedBy = get(cozyContact, 'cozyMetadata.updatedByApps[0].slug')
```

Comment:
I don't think we should rely on the order (even if it should be OK). Maybe something like :

```
get(cozyContact, 'cozyMetadata.updatedByApps', []).reduce(
  (acc, current) => current.date > acc.date ? current : acc,
  { date: null, slug: 'Contacts' }
).slug
```

or `_.maxBy` https://lodash.com/docs/4.17.11#maxBy

Reply:
I almost did it, then figured that we *know* the connector inserts itself in the first position. But if this ever changes we'rll have this problem again, so I'll add the date check!